lidar_point_pillars/CMakeLists.txt

```diff
@@ -55,22 +55,37 @@ else()
   set(CUDNN_AVAIL OFF)
 endif()
 
-if(TRT_AVAIL AND CUDA_AVAIL AND CUDNN_AVAIL)
+# Check TVM utility, runtime libraries and TVM flag
+option(TVM_AVAIL "TVM implementation available" OFF)
```

Comment:
If I'm understanding correctly, activating this flag just essentially means "I built the model using TVM and the output files from TVM are available in the source directory," correct? If so, I would prefer instead that we automatically detect whether the files are available with CMake and set a variable accordingly. e.g.

```
set(TVM_AVAIL false)

file(GLOB TVM_PFE_FILES
  RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}/tvm_models/tvm_point_pillars_pfe/
  "${CMAKE_CURRENT_SOURCE_DIR}/tvm_models/tvm_point_pillars_pfe/*")
file(GLOB TVM_RPN_FILES
  RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}/tvm_models/tvm_point_pillars_rpn/ 
  "${CMAKE_CURRENT_SOURCE_DIR}/tvm_models/tvm_point_pillars_rpn/*")

if ("deploy_graph.json"  IN LIST ${TVM_PFE_FILES} AND
  "deploy_params.params" IN LIST ${TVM_PFE_FILES} AND
  "deploy_lib.so" IN LIST ${TVM_PFE_FILES} AND
  "inference_engine_tvm_config.hpp" IN LIST ${TVM_PFE_FILES} AND
  "deploy_graph.json"  IN LIST ${TVM_RPN_FILES} AND
  "deploy_params.params" IN LIST ${TVM_RPN_FILES} AND
  "deploy_lib.so" IN LIST ${TVM_RPN_FILES} AND
  "inference_engine_tvm_config.hpp" IN LIST ${TVM_RPN_FILES})
  set(TVM_AVAIL true)
endif()
```

Reply:
Done. `IN_LIST`has been introduced in a more recent version of cmake than the one we support here, so I am using `EXISTS` instead.

Reply:
Thanks!